component_documentation/componentDoc.html

```diff
@@ -497,6 +497,363 @@ <h2 class="sub-component__heading">Button with Link</h2>
+                    </div>
+                    <div class="card--icon">
+                        <img src="/image/card_image/like.png" alt="like" class="icon__img">
+                        <img src="/image/card_image/like.png" alt="like" class="icon__img">
+                        <img src="/image/card_image/like.png" alt="like" class="icon__img">
+                       </div>
+                </div>
+            </div>
+           </div>
+           <div class="component-code">
+               <div class="code-snippet-container ">
+                   <textarea class="code-snippet" name="code-snippet" rows="5" cols="" readonly>
+<div class="card">
+    <img src="/image/card_image/river.jpg" alt="Boat and river" class="card__img">
+        <div class="card__title-box">
+            <h2 class="card__heading">Boat & the River</h2>
+            <p class="card__sub-title">by Shreyash Mahajan</p>
+        </div>
+            <p class="card--content">When boat does not know where to go it just floats
+                with the flow of the river.
+             </p>
+         <div class="card__footer">
+                <div class="card__btn-list">
+                    <button class="btnTag btnTag--card">READ</button>
+                        <button class="btnTag btnTag--card" >SAVE</button>
+                    </div>
+                <div class="card--icon">
+                     <img src="/image/card_image/like.png" alt="like" class="icon__img">
+                     <img src="/image/card_image/like.png" alt="like" class="icon__img">
+                     <img src="/image/card_image/like.png" alt="like" class="icon__img">
+                 </div>
+            </div>
+  </div>
+                   </textarea>
+                   <button class="code-snippet--copy">Copy</button>
+                    </div>
+           </div>
+        </div>
+     
+        <!-- card with shadow -->
+        <div class="sub-component-container">
+            <h2 class="sub-component__heading">Card vertical with shadow</h2>
+           <div class="component-display">
+
+            <div class="card card--shadow">
+                <img src="/image/card_image/river.jpg" alt="Boat and river" class="card__img">
+                <div class="card__title-box">
+                <h2 class="card__heading">Boat & the River</h2>
+                <p class="card__sub-title">by Shreyash Mahajan</p>
+                </div>
+                <p class="card--content">When boat does not know where to go it just floats
+                    with the flow of the river.
+                </p>
+                <div class="card__footer">
+                    <div class="card__btn-list">
+                        <button class="btnTag btnTag--card">READ</button>
+                        <button class="btnTag btnTag--card" >SAVE</button>
+                    </div>
+                    <div class="card--icon">
+                        <img src="/image/card_image/like.png" alt="like" class="icon__img">
```

Comment:
Instead of adding the same icon you can use other icons which you have used later such as notify and settings one instead to improve appearance.